app/controllers/admin/appointment_holds_controller.rb

```diff
@@ -17,7 +17,7 @@ def destroy
     def remove_appointment_hold
       @remove_appointment_hold ||= appointment.appointment_holds.find_by(hold_id: params[:id]).destroy
 
-      if params[:cancel_hold]
+      if params[:cancel_hold].to_s.downcase == "true"
```

Comment:
I think this check is OK! The `downcase` is probably not necessary, but it's not going to hurt anything.